From: Robert Mader <robert.mader@posteo.de>
Date: Tue, 20 Dec 2022 20:36:51 +0100
Subject: arm64: dts: rk3399-pinephone-pro: Add camera orientations

Marking the cameras as front and back cameras. This allows downstream to
better name the devices (e.g. "Build-in Front Camera"), selecting
appropriate cameras depending on use-cases and apply e.g. mirroring for
front cameras.

Signed-off-by: Robert Mader <robert.mader@posteo.de>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 7bef501..4b27117 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -870,6 +870,7 @@ wcam: camera@1a {
 		reset-gpios = <&gpio1 RK_PA0 GPIO_ACTIVE_LOW>;
 		powerdown-gpios = <&gpio2 RK_PD4 GPIO_ACTIVE_HIGH>;
 
+		orientation = <1>;
 		rotation = <270>;
 
 		lens-focus = <&wcam_lens>;
@@ -910,6 +911,7 @@ ucam: camera@36 {
 		rockchip,camera-module-name = "CameraKing";
 		rockchip,camera-module-lens-name = "Largan-9569A2";
 
+		orientation = <0>;
 		rotation = <90>;
 
 		port {
